modules/single/hash_demultiplexing.nf

```diff
@@ -14,7 +14,9 @@ include { bff_hashing } from './hash_demulti/bff'
 process summary{
     publishDir "$projectDir/$params.outdir/$params.mode/hash_demulti", mode: 'copy'
     label 'small_mem'
-    label 'summary'
+
+    conda "python=3.9 'pandas<2.0' scanpy muon numpy"
```

Comment:
```suggestion
    conda "python=3.9 'pandas<2.0' scanpy muon"
```

numpy should already be included in scanpy.

Is it a lot of work to make this pandas 2.x compatible? That'd be great together with increasing the Python version to 3.11